api/src/indigo_loaders.cpp

```diff
@@ -36,6 +39,40 @@
 
 #include <limits>
 
+/*
+IndigoJSONLoader::IndigoJSONLoader( Scanner& scanner ) : IndigoObject( JSON_LOADER )
+{
+    json_loader.reset( new JSONLoader( scanner ));
+}
+
+IndigoJSONLoader::IndigoJSONLoader(const char* filename) : IndigoObject(JSON_LOADER)
+{
+    _own_scanner.reset(new FileScanner(indigoGetInstance().filename_encoding, filename));
+    json_loader.reset(new JSONLoader(_own_scanner.ref()));
+}
+
+IndigoJSONLoader::~IndigoJSONLoader()
+{
+}
+
+IndigoObject* IndigoJSONLoader::next()
+{
+    if( !json_loader->hasNext() ) return NULL;
+    return new IndigoJSONMolecule( json_loader->next(), json_loader->rgroups(), json_loader->currentNumber() );
+}
+
+IndigoObject* IndigoJSONLoader::at(int index)
+{
+    if( json_loader->currentNumber() >= index ) return NULL;
+    return new IndigoJSONMolecule( json_loader->at( index ), json_loader->rgroups(), json_loader->currentNumber() );
+}
+
+bool IndigoJSONLoader::hasNext()
+{
+    return json_loader->hasNext();
+}
+*/
+

```

Comment:
Do we need this commented code here?

Reply:
And finally, it turns out that this file has no essential change.